src/__tests__/imagings/Imagings.test.tsx

```diff
@@ -39,31 +39,29 @@ describe('Imagings', () => {
       },
     } as any)
 
-    const wrapper = mount(
+    return render(
       <Provider store={store}>
         <MemoryRouter initialEntries={['/imaging/new']}>
           <TitleProvider>{isNew ? <NewImagingRequest /> : <Imagings />}</TitleProvider>
         </MemoryRouter>
       </Provider>,
     )
-    wrapper.update()
-    return { wrapper: wrapper as ReactWrapper }
   }
 
   describe('routing', () => {
     describe('/imaging/new', () => {
       it('should render the new imaging request screen when /imaging/new is accessed', async () => {
         const permissions: Permissions[] = [Permissions.RequestImaging]
-        const { wrapper } = setup(permissions, true)
+        const { container } = setup(permissions, true)
 
-        expect(wrapper.find(NewImagingRequest)).toHaveLength(1)
+        expect(container).toMatchSnapshot()
       })
 
       it('should not navigate to /imagings/new if the user does not have RequestImaging permissions', async () => {
         const permissions: Permissions[] = []
-        const { wrapper } = setup(permissions)
+        const { container } = setup(permissions)
 
-        expect(wrapper.find(NewImagingRequest)).toHaveLength(0)
+        expect(container).toMatchInlineSnapshot(`<div />`)
```

Comment:
I think this might be a temporary means for removing enzyme, however, I think we should try to avoid snapshot tests wherever possible.

Reply:
`toMatchInlineSnapshot `'s are a little different than the ones that create comparison folders. The general sentiment in RTL is to avoid snapshots at all costs, however, there are cases where they are conducive and even necessary, so if the DOM is completely inaccessible but is an expected state, this is one of the few ways to assert it is actually what it should be. Loaders, some error messages etc... are other cases where sometimes `InlineSnaps` are useful

If you however strongly feel that way we can remove the Inlines and just add an `inTheDocument` check on the container object.

Reply:
If you're interested 😄 This is part of the basis for RTL snapshot philosophy https://kentcdodds.com/blog/effective-snapshot-testing